charts/ztka/templates/configmap-paralus.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 data:
   DEV: "false"
   AUDIT_LOG_FILE: "/audit-logs/audit.log"
+  AUDIT_LOG_STORAGE: {{ include "ztka.auditlogStorage" . | quote }}
```

Comment:
Why is the use of "AUDIT_LOG_STORAGE" variable in Paralus?

Reply:
Based on this variable (postgres/elasticsearch) we will push auditLogs to respective db

Reply:
Got it.